src/plugins/content_management/common/rpc.ts

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
This file is copy paste from the server-side POC https://github.com/elastic/kibana/pull/148791/. 

Assuming this won't change much when initial server-side is merged to main. Also won't be a problem if this is changed a lot